### PR TITLE
New packages: mlst, any2fasta, perl-carp, perl-class-method-modifiers, perl-role-tiny, perl-moo, perl-sub-quote

### PR DESCRIPTION
**Pull request by luke-dt**

### CODE REVIEW

File: var/spack/repos/builtin/packages/any2fasta/package.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/any2fasta/package.py]:
```suggestion
# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

**Review by tldahlgren — CHANGES_REQUESTED** [review]
Confirmed sha256 for the 7 packages, 9 versions.

The copyright for the first package needs to be corrected.

**Comment by luke-dt** [comment]
have I addressed all the problems, @tldahlgren ?

**Review by tldahlgren — APPROVED** [review]
Reconfirmed the sha256.  Thanks for the copyright fix.

**Pull request merged**